### PR TITLE
Include themeable design tokens in `@tokens` to enable autocomplete #DS-1493

### PR DESCRIPTION
**Pull request by adamkudrna**

### CODE REVIEW

File: packages/design-tokens/README.md [hunk 1]
```diff
@@ -79,16 +87,40 @@ npm install --save @lmc-eu/spirit-design-tokens
...
+// vite.config.js
+
+// …
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  css: {
+    preprocessorOptions: {
+      scss: {
+        includePaths: [path.resolve(__dirname, 'node_modules/@lmc-eu/spirit-design-tokens/scss')],
```
Comment (crishpeen) [inline packages/design-tokens/README.md:115]: loadpaths?

File: packages/design-tokens/README.md [hunk 2]
```diff
@@ -243,8 +249,9 @@ With `@` prefix:
 ```scss
 // _Button.scss
 
-@use 'tools'; // Calls './_tools.scss'.
-@use '@global-tokens' as global-tokens; // OK, './_@global.scss' is not here, but the at-sign
+@use 'tools';
+// Calls './_tools.scss'.
```
Comment (crishpeen) [inline packages/design-tokens/README.md:253]: why new line? Prettier?

File: packages/design-tokens/README.md [hunk 3]
```diff
@@ -243,8 +249,9 @@ With `@` prefix:
...
+@use '@tokens' as tokens; // OK, './_@global.scss' is not here, but the at-sign
```
Comment (crishpeen) [inline packages/design-tokens/README.md:254]: global or tokens?

File: packages/web/CONTRIBUTING.md [hunk 4]
```diff
@@ -218,10 +215,10 @@ To avoid conflicts, we need a convention to distinguish situations when somethin
 </details>
 
 [design-tokens]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens
-[tokens-api]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens#tokens-api
-[configuring-load-path]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens#configuring-load-path
-[rebranding]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web#rebranding
+[design-tokens-in-sass]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens#in-sass
+[design-tokens-load-path]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens#in-sass
```
Comment (crishpeen) [inline packages/web/CONTRIBUTING.md:219]: why the same URL? Can we have the link only once?

**Review by crishpeen — APPROVED** [review]

File: packages/design-tokens/README.md (hunk 1 above)
Comment (adamkudrna) [reply packages/design-tokens/README.md:115]: Depends if #1721 will be merged sooner 🙂.

File: packages/design-tokens/README.md [hunk 5]
```diff
@@ -43,21 +45,27 @@ These tokens are shared globally and independently on themes.
...
+Both themes can be used anywhere on the same page and combined as needed.
 
-- Light
-- Light Inverted
+### Using Themes
 
-Where the former is the default and both of them can be used on the same page.
+The `scss` directory contains `@themes.scss` linking all available themes as Sass variables.
+From the technical point of view, the theming is based on CSS variables. **However, this package
+does not provide the CSS variables directly at the moment.** Instead, you need to use the `@themes`
```
[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/design-tokens/README.md:66]: I will change it for you 🙂.

File: packages/design-tokens/README.md [hunk 6]
```diff
@@ -43,21 +45,27 @@ These tokens are shared globally and independently on themes.
...
+file to generate CSS variables. The [`spirit-web`][web-docs] package does this for you and can
+serve as an example of how to do it.
```
[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/design-tokens/README.md:68]: And now?

File: packages/design-tokens/README.md (hunk 1 above)
Comment (crishpeen) [reply packages/design-tokens/README.md:114]: it will definitely be merged first because it will be in `main`.

**Review by crishpeen — APPROVED** [review]

**Review by pavelklibani — APPROVED** [review]

**Pull request merged**